packages/rebber/dist/index.js

```diff
@@ -13,6 +14,8 @@ module.exports.toLaTeX = toLaTeX;
 function toLaTeX(tree) {
   var options = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {};
 
+  options.definitions = definitions(tree, options);
```

Comment:
I didn't know this one, nice catch !

Reply:
Stumbled upon this by chance: https://github.com/syntax-tree/mdast-util-to-hast/blob/742a78036ff62895111e020034fb5225916d94a5/lib/handlers/link-reference.js#L11 :)